### ISSUE TITLE
saveContact no code for writing image

### ISSUE THREAD

**Issue by elvisgraho** [capacitor-community/contacts#59]
Documentation and typescript mention that method saveContact includes image, and that you can set contact image. However native code (android and iOS) has no such feature.

**Comment by Idrimi** [comment]
@elvisgraho true. Function was never implemented. @muuvmuuv is this a Feature you planning to implement or shall i remove it from the docs? I think the feature would be rather large, my tendence is to not implement it.

**Comment by muuvmuuv** [comment]
Oh, you are correct. I just copied ported the CNMutableContact 1:1. Might not be the only prop I forgot to clear afterwards, sorry. You can remove it @Idrimi, I won't have time or the need to implement it.

**Comment by elvisgraho** [comment]
Thank you @muuvmuuv for implementing the saveContact feature. I hope someone in the future can finish with the image.

**Comment by muuvmuuv** [comment]
@elvisgraho if a customer of us wants it we will implement it 👨‍💻. Lets just rename the issue to a feature request and keep it open.

**Comment by AnastasiosF** [comment]
Any news?

**Comment by Trino62** [comment]
I found this example where the contact is saved with an image in android, maybe it can be a starting point, I would like to help but my java knowledge is little
[https://gist.github.com/slightfoot/5985900](url)
